### PR TITLE
Fix minor CodeFactor style issues in ModuleCmdletBase

### PR DESCRIPTION
**Pull request by RDIL**
# PR Summary
Issue: `"A single-line comment must not be followed by a blank line."`

## PR Context

Issue was introduced by Ilya about a year ago according to CodeFactor.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

**Review by iSazonov — APPROVED** [review]

File: src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs [hunk 1]
```diff
@@ -1495,8 +1493,10 @@ private IEnumerable<PSModuleInfo> CreateFakeModuleObject(IEnumerable<ModuleSpeci
 
             string rootModule = null;
             if (
-                !GetScalarFromData<string>(data, moduleManifestPath, "RootModule", manifestProcessingFlags,
-                    out rootModule))
+                !GetScalarFromData<string>(
```
Comment (RDIL) [inline src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs]: this was just my OCD going crazy sorry

Comment (iSazonov) [reply src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs]: Could you please make this in one line?
```c#
!GetScalarFromData<string>(data, moduleManifestPath, "RootModule", manifestProcessingFlags, out rootModule))

Comment (RDIL) [reply src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs]: I really think it looks better like this but sure

**Review by iSazonov — APPROVED** [review]

**Comment by adityapatwardhan** [comment]
Close and re-open to trigger CodeFactor

**Comment by adityapatwardhan** [comment]
@PoshChan please remind me in 1 hour

**Comment by PoshChan** [comment]
@adityapatwardhan, this is the reminder you requested 1 hour ago

**Review by adityapatwardhan — APPROVED** [review]

**Comment by adityapatwardhan** [comment]
@RDIL Thank you for your contribution!

**Pull request merged**